lib/rack/attack/railtie.rb

```diff
@@ -3,17 +3,9 @@
 module Rack
   class Attack
     class Railtie < ::Rails::Railtie
-      initializer 'rack.attack.middleware', after: :load_config_initializers, before: :build_middleware_stack do |app|
+      initializer "rack-attack.middleware" do |app|
         if Gem::Version.new(::Rails::VERSION::STRING) >= Gem::Version.new("5.1")
```

Comment:
With new implementation, I think this restriction can be removed.

Reply:
Probably.

I wan't this change to be the minimal possible update that fixes the issue, which can be released as a patch-level update (`v6.2.1`).

If we open this up for more rails version, that can be part of `v6.3.0`.

Thanks for reviewing.